lib/cnb/base.rb

```diff
@@ -13,7 +13,7 @@ def rate_for(currency, date)
 
       fail CurrencyNotSupported if rate.nil?
 
-      rate.attr('kurz').tr(',', '.').to_f
+      rate.attr('kurz').tr(',', '.').to_f / rate.attr('mnozstvi').to_f
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.